src/web/backend/nextcloud/utils.py

```diff
@@ -0,0 +1,41 @@
+
+
+def edap_to_dict(obj):
+    """ Transform edap object from tuple to dict """
+    return {
+        'fqdn': obj[0],
+        'data': obj[1]
+    }
+
+
+def create_group_folder(nxc, group_name, group_type):
+    """
+    Create group folder for given group_name in group_type
+
+    If folder for group_type doesn't exist - create folder with group_type name first and grant access to group_name,
+    otherwise just give access to group_name, create subfolder in folder for group_type with group_name
+    Args:
+        nxc (NextCloud): nextcloud instance
+        group_name (str): group name
+        group_type (str): group type
+
+    Returns (bool):
+    """
+    # check if folder for type is already created
+    group_folders = nxc.get_group_folders().data
```

Comment:
Please make this a separate method and remove the comment (e.g. `get_group_folder(group_type)`).

Reply:
done